mpas_analysis/ocean/streamfunction_moc.py

```diff
@@ -539,7 +809,7 @@ def _compute_moc_time_series_postprocess(self):  # {{{
 
         latAtlantic = self.lat['Atlantic']
         dLat = latAtlantic - 26.5
-        indlat26 = np.where(dLat == np.amin(np.abs(dLat)))
+        indlat26 = np.where(np.abs(dLat) == np.amin(np.abs(dLat)))
```

Comment:
I believe this was a bug, and we were just lucky that it worked before.

Reply:
I agree.